src/analysis/bounds.rs

```diff
@@ -44,14 +49,23 @@ impl Bounds {
             _ => None,
         }
     }
-    pub fn to_glib_extra(library: &Library, type_id: TypeId) -> String {
+    pub fn to_glib_extra(library: &Library, type_id: TypeId, is_nullable: bool) -> String {
```

Comment:
> @GuillaumeGomez While I prefer library::Nullable over bool I can merge as is.

Then I'll update in this way.

> You don't want squash

I can, just tell me which commits you want me to keep.

> add Into for other options?

Not for now. :P